onadata/settings/common.py

```diff
@@ -92,8 +92,12 @@
 STATIC_URL = '/static/'
 
 # Enketo URL
-ENKETO_API_URL_PARTIAL= os.environ.get('ENKETO_API_URL_PARTIAL', '/api_v1')
-ENKETO_SURVEY_URL_PARTIAL= os.environ.get('ENKETO_SURVEY_URL_PARTIAL', '/survey')
+ENKETO_API_URL_PARTIAL= os.environ.get('ENKETO_API_URL_PARTIAL', '/api_v1') # FIXME: This is a very ugly variable name...
```

Comment:
@dorey, are we still using API v1 anywhere, or do you think we can go ahead and change the default here to "/api/v2"?